js/login.js

```diff
@@ -0,0 +1,105 @@
+const loginEmail = document.getElementById("login-email");
+const emailAlert = document.getElementById("login-email-alert");
+const loginPassword = document.getElementById("login-password");
+const passwordAlert = document.getElementById("login-password-alert");
+const loginBtn = document.getElementById("login-submit");
+const toggleBtn = document.getElementById("toggle-password");
+const eyeIcon = document.getElementById("eye-icon");
+
+// 로그인 버튼 상태 업데이트 함수
+const loginBtnUpdate = () => {
+  // loginBtn이 disabled 되는 조건
+  const isEmailEmpty = loginEmail.value.trim() === "";
+  const isPasswordEmpty = loginPassword.value.trim() === "";
+  const isEmailError = !emailAlert.classList.contains("hidden");
+  const isPasswordError = !passwordAlert.classList.contains("hidden");
```

Comment:
💊 제안
각 Input 값이 비어있으면 error가 존재할테니 아래처럼 확인해도 될 것 같습니다~
```suggestion
  const isEmailError = !emailAlert.classList.contains("hidden");
  const isPasswordError = !passwordAlert.classList.contains("hidden");
```